ghmirror/app/__init__.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import requests
+
+import flask
+
+from ghmirror.app.response import MirrorResponse
+from ghmirror.data_structures.cache import RequestsCache
+from ghmirror.data_structures.cache import StatsCache
+
+
+logging.basicConfig(level=logging.INFO,
```

Comment:
I don't know how much use of kibana we do for our apps so this may be not necessary for now, but structured json logging would be nice

Reply:
Agreed. Let's keep that in mind for future improvements.